great-idea-website/index.html

```diff
@@ -1,63 +1,110 @@
-  <link rel="stylesheet" href="css/index.css">
-
-  <!--[if lt IE 9]>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
-  <![endif]-->
-</head>
+    <!--[if lt IE 9]>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+    <![endif]-->
+  </head>
 
   <body>
 
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
-
-  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
+    <header>
 
-  Get Started
+      <nav>
 
-  <img src="img/header-img.png" alt="Image of a code snippet.">
+        <a href="Services">Services</a>
+        <a href="Product">Product</a>
+        <a href="Vision">Vision</a>
+        <a href="Features">Features</a>
+        <a href="About">About</a>
+        <a href="Contact">Contact</a>
+		
+		
 
-  Features
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+      </nav>
 
+      <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
 
-  About
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
-  
-  Services
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+    </header>
+    
+    <section class="masthead">
+      
+      <h1>
+      Innovation<br>
+      On<br>
+      Demand<br>
+      <button href="#">Get Started</button>
+      </h1>
 
+      
+      
+      <img src="img/header-img.png" alt="Image of a code snippet.">
 
-  Product
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+      
 
-  Vision
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  Contact
-  123 Way 456 Street
-  Somewhere, USA
-  1 (888) 888-8888
-  sales@greatidea.io
-
-  Copyright Great Idea! 2018
+    </section>
   
+    <section class="main-content">
+      
+      <div class="about">
+        <div>
+          <h2>Features</h2>
+      
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus sc elerisque quis.</p>
```

Comment:
Make your p tag a sibling to its h2 tag to keep your code clean. This will make it very clear what the div contains. See line 80, etc. You did it there! :D